frontend/src/composables/useNormRender.ts

```diff
@@ -7,12 +7,14 @@ import { UseFetchReturn } from "@vueuse/core"
  *
  * @param normXml XML of the norm that should be rendered
  * @param showMetadata Enable or disable metadata list at the beginning of the document
+ * @param renderHtml By default (true) renders HTML. When false then XML.
  * @param customNorms The XMLs of norms which are referenced by the norm (e.g. in passiveModifications) and should be used instead of the data stored.
  * @param at Passive modifications coming into effect before this date should be applied before rendering the HTML
  */
 export function useNormRender(
   normXml: MaybeRefOrGetter<string | undefined>,
   showMetadata: MaybeRefOrGetter<boolean> = false,
+  renderHtml: MaybeRefOrGetter<boolean> = true,
```

Comment:
I'm unsure if this additional flag is the best solution here - or if we want to just include a second method. Also with this many parameters (and multiple once that are booleans) I would find it nice if we could move to an options object for all the optional parameters.

Reply:
Alright! I wavered between the two options. I removed the parameter and introduced a second method instead.

If this is fine, I'll provide the missing FE & BE tests on Monday.